common/include/common/common_utils.hpp

```diff
@@ -860,6 +1008,24 @@ static inline void fill_trsm_matrix(std::vector<scalar_t>& A, size_t k,
   }
 }
 
+/**
+ * @breif Fills fully a symmetrical matrix suitable for testing
+ * @param M The matrix to fill. Size must be at least lda * lda
+ * @param lda The leading dimension of matrix @p M
+ */
+template <typename scalar_t>
+static inline void fill_sym_matrix(std::vector<scalar_t>& m, const int n) {
```

Comment:
Maybe, I'm missing something, but I don't get the purpose this function.

What are the features that `m` needs to have?

Reply:
If a symmetric random matrix is not needed, I suggest removing it and generating a random matrix as usual.